etc/profile-a-l/kazam.profile

```diff
@@ -0,0 +1,53 @@
+# Firejail profile for kazam
+# Description: Screen capture tool
+# This file is overwritten after every install/update
+# Persistent local customizations
+include kazam.local
+# Persistent global definitions
+include globals.local
+
+ignore noexec ${HOME}
+
+noblacklist ${PICTURES}
+noblacklist ${VIDEOS}
+noblacklist ${HOME}/.config/kazam
+
+include allow-python2.inc 
+include allow-python3.inc 
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc 
+include disable-interpreters.inc
+include disable-programs.inc
+include disable-passwdmgr.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+include whitelist-runuser-common.inc 
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+net none
+nodvd
+no3d
```

Comment:
Do it need hw-accel?

Reply:
Recording screen needs HW? Also do you need to be part of audio group to record audio? Then `nogroups` needs to be removed.

Reply:
> Recording screen needs HW?

If the record is encode as e.g. mp4.

> Also do you need to be part of audio group to record audio? Then nogroups needs to be removed.

This is usually done with pulse.

Reply:
I'll remove `no3d`.